react/app-admin/constants/api-types.ts

```diff
@@ -132,3 +132,14 @@ export interface APISearchTagStat {
     tag_id: string;
     tag: APITag;
 }
+
+export interface APIGeneralAdminSearchResult {
+    id: string,
+    title: string,
+}
+
+export interface APIGeneralAdminSearch {
+    name: string,
+    slug: string,
+    results: [APIGeneralAdminSearchResult]
```

Comment:
Shouldn't it be an array of `APIGeneralAdminSearchResult`? (`[x]` means it's an array with only one element of type `x`)
```suggestion
    results: APIGeneralAdminSearchResult[]
```

Reply:
Both works but I just changed for consistency !